src/pages/edit_resource/contents_step/AdditionalFilesUpload.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { FC, useEffect, useState } from 'react';
+import UppyDashboard from '../../../components/UppyDashboard';
+import { Uppy } from '../../../types/file.types';
+import { Content } from '../../../types/content.types';
+import { useFormikContext } from 'formik';
+import { ResourceWrapper } from '../../../types/resource.types';
+import { StyledContentWrapper, StyledSchemaPartColored } from '../../../components/styled/Wrappers';
+import placeholderImage from '../../../resources/images/placeholder.png';
+import { UppyFile } from '@uppy/core';
+import LinearProgress from '@material-ui/core/LinearProgress';
+import { Colors } from '../../../themes/mainTheme';
+import styled from 'styled-components';
+import { Typography } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+
+interface AdditionalFilesUploadProps {
+  additionalFileUploadUppy: Uppy;
+  newContent: Content | undefined;
+}
+interface UploadPerFile {
+  percentage: number;
+  filename: string;
+  fileSize?: string;
+  fileType?: string;
+}
+
+const LargeParagraphSpace = styled.div`
+  margin-top: 40px;
+  display: flex;
+  align-items: flex-end;
+  flex-wrap: wrap;
+`;
+
+const SmallParagraphSpace = styled.div`
+  min-width: 200px;
+  display: block;
+  @media (min-width: ${({ theme }) => theme.breakpoints.values.sm + 'px'}) {
+    padding-left: 2rem;
+  }
+`;
+
+const StyledImg = styled.img`
+  width: 100px;
+`;
+
+const UploadImageProgressCard = styled.div`
+  width: 100px;
+`;
+
+const filterAdditionalFiles = (contents: undefined | Content[]) => {
+  if (contents) {
+    return contents.slice(1).filter((content) => content.features.dlr_content_type === 'file') ?? [];
+  } else {
+    return [];
+  }
+};
+
+//Benytter Si enhet for Mega, Giga og Kilo som er 1000 og ikke 1024.
```

Comment:
bare lurer på hvorfor ?  Er det bare fordi jeg er datamann jeg synes det blir feil ? :-)

Reply:
Wait, gjorde det for å glede windows brukere, slik at filstørrelsen blir å matche det som står i filebrowseren, men ser at windows har implementert Kib, Mib, etc. Så da er det jo best at vi bruker det også